src/Manager.php

```diff
@@ -44,6 +44,11 @@ class Manager
     protected $blacklistEnabled = true;
 
     /**
+     * @var bool
+     */
+    protected $retrieveUserOnAuthentication = true;
```

Comment:
I don't think this should be on the `Manager` class, as it is authentication specific, and the Manager has no knowledge or dealings with auth mechanisms.

Instead it should go on the `JWTAuth` class